test/common/testlib.py

```diff
@@ -241,6 +241,11 @@ def open(self, href: str, cookie: Optional[Dict[str, str]] = None, tls: bool = F
             # Force a reload in this case, to make tests and the waitPageLoad below predictable
             # But that option has the inverse effect with Chromium (argh)
             opts["transitionType"] = "reload"
+        elif self.cdp.browser.name == 'chromium':
+            # Chromium also optimizes this away, but doesn't have a knob to force loading
+            # so load the blank page first
+            self.cdp.invoke("Page.navigate", url="about:blank")
+            self.cdp.invoke("waitPageLoad", timeout=5)
```

Comment:
So, I was thinking... can we predict whether the reload will be optimized away? Could we then skip the `waitPageLoad`?

Reply:
Possibly, yes -- then we could skip  both steps. I still want to optimize this, but #1 priority is to get main back to green, so I landed this.